Projects/Claims/Sources/Models/FlowClamSingleItemStepModel.swift

```diff
@@ -115,6 +116,26 @@ public struct FlowClamSingleItemStepModel: FlowClaimStepModel {
     func shouldShowListOfModels(for brand: ClaimFlowItemBrandOptionModel) -> Bool {
         return !(self.getListOfModels(for: brand.itemBrandId)?.isEmpty ?? true)
     }
+
+    var returnDisplayStringForSummary: String {
+        var stringToReturn = ""
+
```

Comment:
you could do it like following:
var textParts:[String] = []
if let purchaseDate {
let purchaseDateText = L10n.summaryPurchaseDateDescription(purchaseDate
textParts.append(purchaseDateText)
}

if let purchasePrice {
let purchasePriceText = L10n.summaryPurchasePriceDescription(Int(purchasePrice)) + " " + (currencyCode ?? "")
textParts.append(purchasePriceText)
}

return textParts.joined(" · ")

Reply:
Fixed it